shanoir-ng-front/src/app/shared/side-menu/side-menu.component.html

```diff
@@ -34,23 +34,17 @@
 
 <ul class="menu-list">
     <li routerLink="/" routerLinkActive="active"><i class="fas fa-home"></i>Home</li>
-    <li (click)="tasksOpened = !tasksOpened" class="notifications">
-        <i class="fas fa-tasks"></i>Tasks
-        <span *ngIf="notificationsService.nbProcess > 0" class="notif">( {{notificationsService.nbProcess}} pending - {{notificationsService.totalProgress() * 100 | number:'0.0-0'}}%)</span>
-        <!-- <button class="awesome" (click)="notificationsService.refresh()" >
-            <i class="fas fa-sync"></i>
-        </button> -->
-        <span class="chevron"><i class="fas fa-chevron-down"></i></span>
+    <li *ngIf="duasToSign() > 0" (click)="notifOpened = !notifOpened"><i class="far fa-flag"></i>

```

Comment:
We need to make an issue about the refactoring of jobs/tasks, to display a new alert for download/import/etc..